oca_dependencies.txt

```diff
@@ -3,3 +3,4 @@ connector
 account-invoicing
 account-financial-tools
 l10n-spain
+contract
```

Comment:
can you put changes to oca_dependencies in a different commit? We need it in a different commit to be able to aggreate all the open PRs without conflicts

Reply:
Yes, I just got a conflict too, @ChrisOForgeFlow can you please split the commit?